packages/proxy/lib/http/response-middleware.ts

```diff
@@ -93,7 +97,7 @@ function resContentTypeIsJavaScript (res: IncomingMessage) {
   )
 }
 
-function isHtml (res: IncomingMessage) {
+function isNotJavascript (res: IncomingMessage) {
   return !resContentTypeIsJavaScript(res)
 }
```

Comment:
Nice cleanup. You could go just a little bit further and make this less confusing, delete this function and update the `isNotJavascript`s to  `isJavascript: resContentTypeIsJavaScript(res)`. The less inversion and indirection the easier it is to understand.

Reply:
Thank you! And agreed. Added that in dcccf74da3295d68621f219bcb6d5b0d7feef880.